docs/src/guide/tutorialmath.jl

```diff
@@ -0,0 +1,271 @@
+# # Mathematical Tutorial
+#
+# This document provides an overview of the structure of DFTK
+# and how to access basic information about calculations.
+# Unlike the [General tutorial](@ref "Tutorial") this document is intended for a mathematical
+# readership, which is not necessarily familiar with materials modelling or
+# plane-wave density-functional theory.
+#
+# ## Density-functional theory models
+#
+# The goal of electronic structure theory is to obtain
+# a quantum-mechanical description of electrons in a material,
+# which is important to accurately predict important physical or chemical properties.
+# Exact electronic structure models are in practice prohibitively expensive,
+# see [Introduction to density-functional theory](@ref) for details.
+# The most widely employed *approximate* electronic-structure model is
+# **Kohn-Sham density-functional theory**, which is the focus in this tutorial.
+#
+# Mathematically, Kohn-Sham DFT is described by an energy minimisation problem,
+# in which the unknowns are functions $\mathbb{R}^3 → \mathbb{C}$, called **orbitals**
+# $ψ_{kn}$ as well as for each orbital an associated occupation value
+# $f_{kn}$, a scalar. These collectively make up the so-called density matrix,
+# which describes the many-body quantum state of the electrons of the material.
+# For details on this energy minimisation problem and its connection to the exact
+# theory, see [Introduction to density-functional theory](@ref).
+#
+# While approaches to directly solve above energy minimisation problem exist,
+# the common approach is to directly satisfy the corresponding first-order optimality
+# conditions, also called the **self-consistent field equations**.
+# These read
+# ```math
+# \tag{1}
+# \left\{
+# \begin{aligned}
+# &\left( -\frac12 (-i∇ + k)^2 + V\left(\rho\right) \right) ψ_{kn}
+#   = ε_{kn} ψ_{kn}, \qquad \text{for } 1 ≤ n ≤ N, k ∈ Ω^∗ ⊂ \mathbb{R}^3 \\
+# V(ρ) = &\, V_\text{nuc} + V_\text{H}^ρ + V_\text{XC}(ρ), \\
+# ρ(r) = & \frac{1}{|Ω|}  ∫_{Ω^∗} ∑_{n=1}^N f(ε_{kn})  \abs{ψ_{kn}(r)}^2 \, d k, \\
+# \end{aligned}
+# \right.
+# ```
+# where for each $k ∈ Ω^∗$ the $ε_{k1} ≤ ε_{k2} ≤ ⋯ ≤ ε_{kN}$
+# are the eigenvalues and the orbitals $ψ_{kn} ∈ H^1(\mathbb{R}^3)$
+# the eigenfunctions of the non-linear operator
+# ```math
+# H = -\frac12 \Delta + V\left(\rho\right)
+# ```
+# making this problem an eigenvector-dependent non-linear eigenvalue problem.
+# Here, $-\frac12 (-i∇ + k)^2$ is the **kinetic operator**.
+# It arises from a quantum-mechanical
+# treatment of the classical kinetic energy of the electrons. In turn, $V$ is the
+# non-linear **potential**, which arises
+# from the interaction of the electrons amongst each other
+# as well as the interactions of the electrons with
+# the nuclei of a material. It can be shown that for the usual potentials
+# (Coulomb or more regular) that $H$ is self-adjoint with compact support, meaning that
+# its spectrum is entirely discrete and its eigenfunctions countable.
```

Comment:
This should probably be qualified to refer to a Bloch fiber H_k (discrete spectrum) acting on $L^2(\Omega)$ instead of the full H (continuous spectrum) acting on $L^2(\mathbb{R}^3)$